src/gridstack-poly.js

```diff
@@ -202,4 +219,46 @@ if (!Array.prototype['forEach']) {
     }
     // 8. return undefined
   };
-}
\ No newline at end of file
+}
+
+// https://developer.mozilla.org/zh-CN/docs/Web/API/CustomEvent/CustomEvent
+(function () {
+  if (window.navigator.userAgent.indexOf('Chrome') > -1 ) {

```

Comment:
did you actually get that from core-js polyfill ? checking for specific browser type seems bridle to me...